src/test/java/org/sagebionetworks/web/unitclient/widget/entity/controller/EntityActionControllerImplTest.java

```diff
@@ -2468,9 +2468,9 @@ public void testOnCreateEntityViewSnapshot() {
     handler.onComplete(mockTableUpdateTransactionResponse);
     verify(mockView).hideCreateVersionDialog();
     verify(mockPopupUtils).notify(any(), any(), any(ToastMessageOptions.class));
+    // EntityUpdatedEvent is fired to invalidate caches
+    verify(mockEventBus).fireEvent(any(EntityUpdatedEvent.class));
     verify(mockPlaceChanger).goTo(any());
-    // Don't fire an updated event; we used placeChanger to go to a new place.
-    verify(mockEventBus, never()).fireEvent(any(EntityUpdatedEvent.class));
```

Comment:
Alternatively we could have placechanger invalidate all caches, but that's pretty heavy-handed

Reply:
Like to avoid this option